packages/tradershub/src/screens/Address/Address.tsx

```diff
@@ -111,7 +113,7 @@ const Address = () => {
                                     />
                                 </div>
                                 <Input
-                                    className='text-body-sm'
+                                    className='text-default'
```

Comment:
If there's any expected styles missing from component please bring it up to the deriv-ui discussion channel so we can issue a fix 😄

Reply:
Will raise it up to ui team